bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/clangd/internal/config/ClangFormatMonitor.java

```diff
@@ -77,7 +77,7 @@ private void createClangFormatFile(IWorkbenchPartReference partRef) {
 			if (file == null) {
 				return;
 			}
-			formatUtils.createClangFormatFile(file.getProject());
+			formatFile.createClangFormatFile(file.getProject());
```

Comment:
Since we use `IWorkbench` inside this class anyway, we can ask workbench for `ClangdFormatFile` service instance when we need it.